src/containers/NamespacesDropdown/NamespacesDropdown.stories.js

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
You'll also need to update the storybook config so this gets loaded: https://github.com/tektoncd/dashboard/blob/15b946eb3334095a6cb52e70c9860e40363afc51/.storybook/config.js#L41

```diff
-const req = require.context('../src/components', true, /\.stories.js$/);
+const req = require.context('../src', true, /\.stories.js$/);
```